spark-doris-connector/src/main/scala/org/apache/doris/spark/load/StreamLoader.scala

```diff
@@ -393,7 +394,8 @@ class StreamLoader(settings: SparkSettings, isStreaming: Boolean) extends Loader
       return null;
     }
     val calendar = Calendar.getInstance
-    "spark_streamload_" +
+    val labelPrefix = streamLoadProps.getOrElse("label_prefix", "spark_streamload")
+    labelPrefix + "_" +
```

Comment:
Would it be better to expose a parameter directly here, such as doris.sink.label-prefix? Adding it directly to streamLoadProps may cause confusion in the streamload header?